examples/detection_callback.py

```diff
@@ -25,11 +25,18 @@ def simple_callback(device: BLEDevice, advertisement_data: AdvertisementData):
 
 
 async def main(service_uuids):
-    if len(service_uuids) > 0 and service_uuids[0] == "all":
-        # in Macos Monterey the service_uuids need to be specified.
+    if sys.platform == "darwin" and service_uuids and service_uuids[0] == "all":
```

Comment:
I would suggest that we enable this by default (without any parameter). And we use the logger.info block (maybe logging.warining would be better) below to give the user an hint that we only scan for predefined uuids.

Reply:
Updated.